franklab_scripts/README.md

```diff
@@ -29,9 +29,8 @@ regularly as cron jobs.
 3. Set up a cron job to run `run_jobs.sh` at the desired interval by running
     `crontab -e` and adding the script.
 
```

Comment:
Do we want to call this franklab scripts or something more general that other labs can repurpose?

Reply:
Happy to rename, maybe `maintenance_scripts`? Is it worth moving these .py files into the package itself and then maybe putting the .sh in the docs, to cut down on the root level folders?

Reply:
That works for me.